### PR TITLE
kobalt-tool-window

### PR DESCRIPTION
**Pull request by dmitry-zhuravlev**
Implemented features:
- Tool window (like Maven, Ant, Gradle...)
- Kobalt task execution from tool window: via double click on corresponding node or toolbar's Run button click

Further adjustments should be:
- Unique icons for nodes, run tab
- Creation of Run configuration

### CODE REVIEW

**Comment by cbeust** [comment]
Wow, this looks awesome, thanks @dmitry-zhuravlev! I had no idea such a PR was in the work :-)

Are you in the #kobalt Slack? Want to join us there to chat?

**Comment by cbeust** [comment]
This change adds the following tool window:

image

which in turn enables:

image

File: src/com/beust/kobalt/intellij/toolWindow/KobaltProjectsTreeNodes.kt [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+        override fun doBuildChildren() = projectNodes
+    }
+
+    class ProjectNode(displayName: String, parent: BaseNode) : BaseNode(displayName, parent) {
+
+        override val actionId = null
+
+        val taskNodes = listOf(
+                TaskNode("clean", this), //TODO better get from server
```
Comment (cbeust) [inline src/com/beust/kobalt/intellij/toolWindow/KobaltProjectsTreeNodes.kt:57]: Indeed. I should implement a new command "GetTasks" that will give you these tasks. But it's perfectly fine to hardcode them in this first version.

**Comment by cbeust** [comment]
Quick question: I'm now seeing the following error in `plugin.xml`:

image

But the id is defined just above. Any thoughts why this is happening?

File: src/com/beust/kobalt/intellij/toolWindow/KobaltProjectsTreeNodes.kt (hunk 1 above)
Comment (dmitry-zhuravlev) [inline src/com/beust/kobalt/intellij/toolWindow/KobaltProjectsTreeNodes.kt:57]: Yes, that's why I put this TODO in the code ;)

**Comment by dmitry-zhuravlev** [comment]
Hi Cedric!

First of all I would like to say thank you for Kobalt itself. I think it is a great idea to have build tool like Gradle but without Groovy. So here is my first start to help with improvement of the Intellij IDEA plugin. 
I will continue to contribute some things to make it better like Maven, Ant, Gradle plugins. 
I hope I'll have a time to do all things needed to have a better integration ;)

Also I can join #kobalt Slack but please give me a hint how to do so :)

**Comment by dmitry-zhuravlev** [comment]
Errors in plugin.xml looks like a validation bug. Created ticket https://youtrack.jetbrains.com/issue/IDEA-155089

**Comment by cbeust** [comment]
Hi Dmitry,

That's great to hear! I definitely need help with the IDEA plug-in so your contributions are very welcome. 

Get an invite to Slack here:  http://kotlinslackin.herokuapp.com/

And then just join the `#kobalt` channel.

Talk to you soon!

**Pull request merged**